loki/README.md

```diff
@@ -0,0 +1,159 @@
+
+1. To install the Loki stack with Kyma's LogPipeline feature based on Fluent Bit, run:
+
+   ```bash
+   helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml --set promtail.enabled=false
+   ```
+
+2. Download the [LogPipeline](logpipeline-custom.yaml) and replace the `{HELM_RELEASE_NAME}` and `{NAMESPACE}` placeholder.
+
+3. Apply the modified LogPipeline:
+
+   ```bash
+   kubectl apply -f logpipeline-custom.yaml
+   ```
+
+When the status of the applied LogPipeline resource turned into `Running`, the underlying Fluent Bit is reconfigured and log shipment to your Loki instance is active.
+
+> **TIP:** The used output plugin configuration uses all labels of a Pod to label the Loki log streams. Performancewise, such segregation of the log streams might be not optimal. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailormade setup that fits your workload configuration.
+
+  </details>
+  <details>
+  <summary label="installation-with-grafana">
+  Installation with Grafana
+  </summary>
+
+  The used Helm chart supports the deployment of Grafana as well, but is disabled by default. Because Grafana provides a very good Loki integration, you might want to install it as well.
+
+  1. To deploy Grafana alongside Loki, with Loki pre-configured as a datasource, run:
+
+     ```bash
+     helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/grafana-values.yaml --set grafana.adminPassword=myPwd
+     ```
+
+  2. To access the Grafana UI with kubectl port forwarding, run:
+
+     ```bash
+     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
+     ```
+
+  3. Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
+  
+  </details>
+</div>
+
+### Verify the installation
+
+Check that the `loki` Pod has been created in the Namespace and is in the `Running` state:
+
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get pod -l app=loki,release=${HELM_RELEASE_NAME}
+```
+
+### Verify the setup by accessing logs using the Loki API
+
+1. To access the Loki API, use kubectl port forwarding. Run:
+
+   ```bash
+   kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/$(kubectl  get svc -n ${KYMA_LOKI_EXAMPLE_NS} -l app=loki,release=${HELM_RELEASE_NAME},variant=headless -ojsonpath='{.items[0].metadata.name}') 3100
+   ```
+
+1. Loki queries need a query parameter `time`, provided in nano seconds. To get the current nano seconds in Linux or MacOS, run:
```

Comment:
```suggestion
1. Loki queries need a query parameter **time**, provided in nanoseconds. To get the current nanoseconds in Linux or macOS, run:
```
According to [our guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-bold-font-for-these-items) we use bolding when we talk about parameters. Unless it's not the case here.